website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -3,8 +3,6 @@ title: "Column-level lineage"
 description: "Use dbt Explorer's column-level lineage to gain insights about your data at a granular level."
 ---
 
-# Column-level lineage <Lifecycle status='public preview' />
-
 dbt Explorer now offers column-level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a resource (model, source, or snapshot) in a dbt project, Explorer provides end-to-end lineage for the data in that column given how it's used.
 
 CLL is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the [Discovery API](/docs/dbt-cloud-apis/discovery-api).
```

Comment:
Please remove "It’s also available through the [Discovery API](/docs/dbt-cloud-apis/discovery-api)."

Reply:
Done! it's included in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5488/commits/6d409449d1b65a3b892200dc1c5a5753c0522eee)